src/main/resources/application-openshift-metering-worker.yaml

```diff
@@ -11,6 +11,7 @@ rhsm-subscriptions:
             #{metric.queryParams[prometheusMetric]}
             * on(_id) group_right
             min_over_time(#{metric.queryParams[prometheusMetadataMetric]}{product="#{metric.queryParams[product]}", ebs_account="#{runtime[account]}", billing_model="marketplace", support=~"Premium|Standard|Self-Support|None"}[1h])
+          #   min_over_time(#{metric.queryParams[prometheusMetadataMetric]}{product="#{metric.queryParams[product]}", ebs_account="#{runtime[account]}", billing_model="marketplace", support=~"Premium|Standard|Self-Support|None|Eval"}[1h])
```

Comment:
I think we should delete this so we don't forget to remove it later

Reply:
Agreed. I put it there for convenience of testing the InstanceAPI for rhosak.

Reply:
Instead of checking in commented-out properties for convenience of testing, you can create a `config/application.yaml` locally in the project and override whatever properties you'd like.  Files in this directory are omitted via the .gitignore file.

```yaml
rhsm-subscriptions:
  metering:
    prometheus:
      metric:
        queryTemplates:
          default: >-
            #{metric.queryParams[prometheusMetric]}
            * on(_id) group_right
            min_over_time(#{metric.queryParams[prometheusMetadataMetric]}{product="#{metric.queryParams[product]}", ebs_account="#{runtime[account]}", billing_model="marketplace", support=~"Premium|Standard|Self-Support|None|Eval"}[1h])
```